pkg/provider/azure_vmssflex_cache.go

```diff
@@ -197,7 +198,7 @@ func (fs *FlexScaleSet) getNodeVmssFlexID(nodeName string) (string, error) {
 	}
 
 	vmssFlexID, err := getter(nodeName, azcache.CacheReadTypeDefault)
-	if err == cloudprovider.InstanceNotFound {
+	if errors.Is(err, cloudprovider.InstanceNotFound) {
```

Comment:
error might be wrapped with some messages, Do you think we need to invoke errors.As instead?

Reply:
to log?